### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by trevorfitz0**
## Necessary check marks:

    [] All Tests are Passing in all environments

    [✓] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [✓] Refactor

## Description of Change:

    description: 
    • Fixed styling to match on all pages
    • Placeholder for input fix
    • Map Use Effect error fixed

## Check the correct boxes

    [✓] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [✓] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [✓] My code has no unused/commented out code
    [✓] My code has no binding.pry's
    [✓] I have reviewed my code
    [✓] I have commented my code, particularly in hard-to-understand areas
    [✓] I have fully tested my code

### CODE REVIEW

File: src/Components/CreateEvent/CreateEvent.js [hunk 1]
```diff
@@ -10,32 +10,40 @@ function CreateEvent() {
...
+    //This is just so the properties are used for now, our API POST will use these variables later
+    console.log(eventName, eventDescription, date, time, group)
+
   return (
     <div>
         <section className='card'>
             <h1 className='title'>Create New Event</h1>
             <input className='long-input'
-                value={eventName}
+                placeholder='Event Name'
```
Comment (jfogiato) [inline src/Components/CreateEvent/CreateEvent.js:21]: Heck yes - looks great!

File: src/Components/GroupPage/GroupPage.css [hunk 2]
```diff
@@ -1,40 +1,84 @@
...
+.short-tile {
+    width: 25vw;
+    height: 5vh;
+    display: flex;
+    justify-content: center;
+    background-color: white;
+}
+
+.selected {
+    color: var(--secondary-color-white);
```
Comment (jfogiato) [inline src/Components/GroupPage/GroupPage.css:41]: WHEW - thanks for all the work on this!

File: src/Components/Map/Map.js [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+    fields: ["formatted_address", "geometry", "name"],
+    strictBounds: false,
+    types: ["establishment"]
+  };
+
+  autoCompleteRef.current = new window.google.maps.places.Autocomplete(
+   inputRef.current,
+   options
+  );
+ }, []);
```
Comment (jfogiato) [inline src/Components/Map/Map.js:19]: Stoked that you got this figured out!! Can't wait to see it in action!

**Review by jfogiato — APPROVED** [review]
Looks great! Thanks for all the work on this!

**Review by Gderibeaux — APPROVED** [review]
Everything looks good!!

**Pull request merged**